spec/controllers/application_controller_spec.rb

```diff
@@ -11,7 +11,7 @@ def index
     it 'sets headers to disable cache' do
       get :index
 
-      expect(response.headers['Cache-Control']).to eq 'private, no-store'
+      expect(response.headers['Cache-Control']).to eq 'no-store'
```

Comment:
Since `private` with `no-store` is superfluous, the `private` segment was removed: https://github.com/rails/rails/pull/39461

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control says:
>**private**
    The response may be stored only by a browser's cache, even if the response is normally non-cacheable. **If you mean to not store the response in any cache, use no-store instead**. This directive is not effective in preventing caches from storing your response.